design/MonitoringModeAPI.md

```diff
@@ -3278,6 +3278,7 @@ The response will contain a array of JSON object with the updated recommendation
 | 400              | experiment_name is mandatory.                                                                      |
 | 400              | interval_end_time is mandatory.                                                                    |
 | 400              | Given timestamp - \" 2023-011-02T00:00:00.000Z \" is not a valid timestamp format.                 |
+| 400              | No metrics available from `2024-01-15T00:00:00.000Z` to `2023-12-31T00:00:00.000Z`.                |
 | 400              | Data not found!.                                                                                   |
```

Comment:
@khansaad - Is Data not found message still displayed or changed now? Can you check if this requires an updated.